### ISSUE TITLE
Resolve golint Suggestions

### ISSUE THREAD

**Issue by elithrar** [cloudflare/cloudflare-go#9]
I'll probably tackle this, but I'll post it here for posterity:

```go
~/.go/src/github.com/jamesog/cloudflare-go(msilverlock/breaking-options ✗) golint
cloudflare.go:49:1: comment on exported function NewZone should be of the form "NewZone ..."
cloudflare.go:199:6: exported type ZoneResponse should have comment or be unexported
cloudflare.go:206:6: exported type ZonePlanResponse should have comment or be unexported
cloudflare.go:227:6: exported type ZoneSetting should have comment or be unexported
cloudflare.go:235:6: exported type ZoneSettingResponse should have comment or be unexported
cloudflare.go:242:1: comment on exported type DNSRecord should be of the form "DNSRecord ..." (with optional leading article)
cloudflare.go:261:1: comment on exported type DNSRecordResponse should be of the form "DNSRecordResponse ..." (with optional leading article)
cloudflare.go:269:1: comment on exported type DNSListResponse should be of the form "DNSListResponse ..." (with optional leading article)
cloudflare.go:277:1: comment on exported type ZoneRailgun should be of the form "ZoneRailgun ..." (with optional leading article)
cloudflare.go:285:6: exported type ZoneRailgunResponse should have comment or be unexported
cloudflare.go:292:1: comment on exported type ZoneCustomSSL should be of the form "ZoneCustomSSL ..." (with optional leading article)
cloudflare.go:308:6: exported type ZoneCustomSSLResponse should have comment or be unexported
cloudflare.go:315:6: exported type KeylessSSL should have comment or be unexported
cloudflare.go:327:6: exported type KeylessSSLResponse should have comment or be unexported
cloudflare.go:334:6: exported type Railgun should have comment or be unexported
cloudflare.go:353:6: exported type RailgunResponse should have comment or be unexported
cloudflare.go:360:1: comment on exported type CustomPage should be of the form "CustomPage ..." (with optional leading article)
cloudflare.go:371:6: exported type CustomPageResponse should have comment or be unexported
cloudflare.go:378:1: comment on exported type WAFPackage should be of the form "WAFPackage ..." (with optional leading article)
cloudflare.go:389:6: exported type WAFPackagesResponse should have comment or be unexported
cloudflare.go:400:6: exported type WAFRule should have comment or be unexported
cloudflare.go:414:6: exported type WAFRulesResponse should have comment or be unexported
cloudflare.go:425:6: exported type PurgeCacheRequest should have comment or be unexported
cloudflare.go:431:6: exported type PurgeCacheResponse should have comment or be unexported
dns.go:9:1: comment on exported method API.CreateDNSRecord should be of the form "CreateDNSRecord ..."
dns.go:38:1: comment on exported method API.DNSRecords should be of the form "DNSRecords ..."
dns.go:81:1: comment on exported method API.DNSRecord should be of the form "DNSRecord ..."
dns.go:108:1: comment on exported method API.UpdateDNSRecord should be of the form "UpdateDNSRecord ..."
dns.go:143:1: comment on exported method API.DeleteDNSRecord should be of the form "DeleteDNSRecord ..."
keyless.go:3:1: comment on exported method API.CreateKeyless should be of the form "CreateKeyless ..."
keyless.go:8:1: comment on exported method API.ListKeyless should be of the form "ListKeyless ..."
keyless.go:13:1: comment on exported method API.Keyless should be of the form "Keyless ..."
keyless.go:18:1: comment on exported method API.UpdateKeyless should be of the form "UpdateKeyless ..."
keyless.go:23:1: comment on exported method API.DeleteKeyless should be of the form "DeleteKeyless ..."
railgun.go:5:1: comment on exported method API.CreateRailgun should be of the form "CreateRailgun ..."
railgun.go:24:1: comment on exported method API.Railguns should be of the form "Railguns ..."
railgun.go:29:1: comment on exported method API.Railgun should be of the form "Railgun ..."
railgun.go:34:1: comment on exported method API.ZoneRailgun should be of the form "ZoneRailgun ..."
ssl.go:3:1: comment on exported method API.CreateSSL should be of the form "CreateSSL ..."
ssl.go:8:1: comment on exported method API.ListSSL should be of the form "ListSSL ..."
ssl.go:13:1: comment on exported method API.SSLDetails should be of the form "SSLDetails ..."
ssl.go:18:1: comment on exported method API.UpdateSSL should be of the form "UpdateSSL ..."
ssl.go:23:1: comment on exported method API.ReprioSSL should be of the form "ReprioSSL ..."
ssl.go:28:1: comment on exported method API.DeleteSSL should be of the form "DeleteSSL ..."
user.go:5:1: comment on exported method API.UserDetails should be of the form "UserDetails ..."
user.go:23:1: comment on exported method API.UpdateUser should be of the form "UpdateUser ..."
waf.go:7:1: exported method API.ListWAFPackages should have comment or be unexported
waf.go:23:10: should omit 2nd value from range; this loop is equivalent to `for pi := range ...`
waf.go:29:1: exported method API.ListWAFRules should have comment or be unexported
waf.go:45:10: should omit 2nd value from range; this loop is equivalent to `for ri := range ...`
zone.go:8:1: comment on exported method API.CreateZone should be of the form "CreateZone ..."
zone.go:17:1: comment on exported method API.ListZones should be of the form "ListZones ..."
zone.go:42:12: should omit 2nd value from range; this loop is equivalent to `for zi := range ...`
zone.go:61:1: comment on exported method API.ZoneDetails should be of the form "ZoneDetails ..."
zone.go:92:1: comment on exported function EditZone should be of the form "EditZone ..."
zone.go:97:1: comment on exported method API.PurgeEverything should be of the form "PurgeEverything ..."
zone.go:112:1: comment on exported method API.PurgeCache should be of the form "PurgeCache ..."
zone.go:127:1: comment on exported function DeleteZone should be of the form "DeleteZone ..."
```

**Comment by jamesog** [comment]
As it happens I'm working on this right now :-)

I've had a WIP branch for a while so I'll finish that off shortly.

**Comment by jamesog** [comment]
Ah, you've included the changes in #10 anyway. That's cool.

**Comment by elithrar** [comment]
@jamesog Just a few! I wanted to get that PR out to discuss the other features/changes first.

**Comment by jamesog** [comment]
I've added you as a collaborator to the project. Here, have your first issue. ;-)

**Comment by elithrar** [comment]
Closed via #10.
